src/io/flutter/run/SdkRunConfig.java

```diff
@@ -83,10 +83,11 @@ public LaunchState getState(@NotNull Executor executor, @NotNull ExecutionEnviro
     final MainFile main = MainFile.verify(launchFields.getFilePath(), env.getProject()).get();
     final Project project = env.getProject();
     final RunMode mode = RunMode.fromEnv(env);
+    final Module module = ModuleUtil.findModuleForFile(main.getFile(), env.getProject());
```

Comment:
Aside: it looks like `MainFile.verify` is sure to return non-null; maybe we should annotate it that way?